protected/views/site/team.php

```diff
@@ -1,10 +1,14 @@
 <div class="content">
     <div class="container">
-        <section>
-            <div class="page-title">
-                <h4>Our team</h4>
-            </div>
-        </section>
+        <div class="page-title">
```

Comment:
for some reason this page did not have breadcrumbs, it seemed a good idea to add them as all other pages do